docs/course_materials/compound_data_types.html

```diff
@@ -176,8 +176,10 @@ <h3><b>In-place list operations</b></h3>
 print(fruit)  # ['plum', 'orange', 'apple']
                     </code></pre>
                 </section>
+                <section data-background-size="cover" data-background-image="media/exr_background.jpeg">
```

Comment:
No need to open a new section for that one.
add the data-background-size="cover" data-background-image="media/exr_background.jpeg" to the exists section right before the data-markdown.